lib/presentation/widget/biometric_authentication/biometric_authentication_widget.dart

```diff
@@ -57,7 +55,9 @@ class _BiometricAuthenticationState extends State<BiometricAuthenticationWidget>
   void initState() {
     super.initState();
     WidgetsBinding.instance.addObserver(this);
-    _biometricAuthenticationViewModel.getBiometricSetting();
+    WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
+      _biometricAuthenticationViewModel.authenticationBiometric(context);
+    });
```

Comment:
Can you say sth about that?

Reply:
I wait until the widget is finished rendering before displaying the biometric authentication dialog.